ledger/db/src/mint_config_store.rs

```diff
@@ -28,8 +28,8 @@ use mc_util_serial::{decode, encode, Message};
 // LMDB Database names.
 pub const ACTIVE_MINT_CONFIGS_BY_TOKEN_ID_DB_NAME: &str =
     "mint_config_store:active_mint_configs_by_token_id";
-pub const BLOCK_INDEX_BY_MINT_CONFIG_TX_NONCE_DB_NAME: &str =
-    "mint_config_store:block_index_by_mint_config_tx_nonce";
+pub const BLOCK_INDEX_BY_MINT_CONFIG_TX_NONCE_AND_TOKEN_ID_DB_NAME: &str =
+    "mint_config_store:block_index_by_mint_config_tx_nonce_and_token_id";
```

Comment:
I think this isn't going to work, we need to actually create a migration don't we? We're going to turn on the nonces in prod soon, we can't just delete the old tables?

Reply:
I attempted the migration, and tested it locally. I think it should be ok to delete the old tables and rebuild in this migration path.

Reply:
okay sounds good to me :)